creme/reports/templates/reports/view_graph.html

```diff
@@ -1,3 +1,5 @@
+{% extends 'creme_core/generics/view_entity.html' %}
```

Comment:
Hum you finally use a classical entity's detail-view ?! Why don't load bricks manually ? (see my remark for populate.py about DB configuration of graph's bricks)

Reply:
Huh ? I've lost some changes here :sob: